<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonEditorPaneClass</class>
 <widget class="QWidget" name="PersonEditorPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PersonEditorPane</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Last name:</string>
     </property>
     <property name="buddy">
      <cstring>lastNameInput</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;First name:</string>
     </property>
     <property name="buddy">
      <cstring>firstNameInput</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cl&amp;ub:</string>
     </property>
     <property name="buddy">
      <cstring>clubInput</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lastNameInput"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="firstNameInput"/>
   </item>
   <item row="3" column="1">
    <widget class="SkComboBox" name="clubInput">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="commentsInput"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Commen&amp;ts:</string>
     </property>
     <property name="buddy">
      <cstring>commentsInput</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Medical &amp;valid until:</string>
     </property>
     <property name="buddy">
      <cstring>medicalValidityInput</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Check &amp;medical:</string>
     </property>
     <property name="buddy">
      <cstring>checkMedicalInput</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="SkComboBox" name="checkMedicalInput">
        <property name="toolTip">
         <string>&lt;html&gt;Check this person's medical validity when creating a new flight&lt;html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="medicalCheckDisabledLabel">
        <property name="toolTip">
         <string>&lt;html&gt;The global medical check option is disabled. For medicals to be checked, enable the option &quot;Check medicals&quot; in the program settings.&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Medical check is disabled!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>148</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QWidget" name="medicalValidityPane" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="medicalValidityUnknownCheckbox">
        <property name="text">
         <string>&amp;Unknown</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="medicalValidityInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;The last date that this person's medical is valid&lt;/html&gt;</string>
        </property>
        <property name="displayFormat">
         <string>MM/dd/yyyy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="displayMedicalValidityButton">
        <property name="text">
         <string>&amp;Display</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Club &amp;ID:</string>
     </property>
     <property name="buddy">
      <cstring>clubIdInput</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="clubIdInput">
     <property name="toolTip">
      <string>&lt;html&gt;This field can be used for a member ID or similar.&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SkComboBox</class>
   <extends>QComboBox</extends>
   <header>src/gui/widgets/SkComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lastNameInput</tabstop>
  <tabstop>firstNameInput</tabstop>
  <tabstop>clubInput</tabstop>
  <tabstop>checkMedicalInput</tabstop>
  <tabstop>medicalValidityUnknownCheckbox</tabstop>
  <tabstop>medicalValidityInput</tabstop>
  <tabstop>clubIdInput</tabstop>
  <tabstop>commentsInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
